packages/demo-cards/claim.gts

```diff
@@ -44,17 +56,27 @@ class Isolated extends Component<typeof Claim> {
   constructor(owner: unknown, args: any) {
     super(owner, args);
     this.initialize.perform();
+    if (this.isMetamaskInstalled) {
+      window.ethereum.on('chainChanged', (chainId: string) => {
+        this.connected =
+          parseInt(chainId, 16) == this.args.model.chain?.chainId;
+        window.location.reload(); // metamask recommends to reload page
```

Comment:
I saw that metamask recommends this so I put it here. This is the only listener I keep here tho there are some possiblities for others.

Reply:
hmmmm, i think this might make this card play nice with others. perhaps if we want something like this we should introduce some kind of API so that cards may request a page reload. @ef4 what do you think?